Tiqr/tiqr-app-core-android
# Bugfix/tiqr 301 change button to allow deny

### Pull Request Checklist

### Short Description of Change
Use Allow/Deny i.s.o. OK/Cancel on the authentication confirmation screen.

### Motivation and Context


### Testing Steps
For an enrolled user, log in.
On the confirm login screen, check the buttons are Deny/Allow vs. Cancel/OK.


### Additional Information
![image](https://user-images.githubusercontent.com/2356050/202490002-64debae3-3bca-4708-9d6e-54b52c12a437.png)
